<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudiPi</class>
 <widget class="QMainWindow" name="AudiPi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudiPi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="playlistWidget"/>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="trackLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="prevButton">
          <property name="text">
           <string>Previous</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::MediaSkipBackward"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playPauseButton">
          <property name="text">
           <string>Play</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextButton">
          <property name="text">
           <string>Next</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::MediaSkipForward"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
